public/compute/first_contact.md

```diff
@@ -1,18 +1,80 @@
-#First contact
-##What is MOLGENIS Compute?
+# What is MOLGENIS Compute?
+
 MOLGENIS Compute is a tool to generate shell script files for big data workflows that can run in parallel on clusters and grids.
 
 The code is open source and hosted on GitHub.
 http://github.com/molgenis/molgenis-compute
 
-#The Black Triangle
+The software is licensed under the LGPL license.
 
-## Download
+# Download
 You can download the [latest version of MOLGENIS Compute](https://github.com/molgenis/molgenis-compute/releases) from GitHub.
 
-## Domain model
-In MOLGENIS Compute, Data is processed using a Workflow that consists of a series of Steps that depend on each other. Each Step follows a Protocol, which is a parameterized template for the shell script for that Step.
+# Some Terms
+In MOLGENIS Compute, **Data** is processed using a **Workflow** that consists of a series of **Steps** that depend on each other. Each **Step** follows a **Protocol**, which is a parameterized **Template** for the shell script for that **Step**.
+
+# Demo
+## Create a workflow
+We assume you have downloaded and unzipped molgenis compute commandline and are now in the directory containing the unzipped files.
+
+You can generate a template for a new workflow using the command:
+
+```bash
+  sh molgenis_compute.sh --create myfirst_workflow
+```
+
+This will create a new directory for the workflow:
+
+```bash
+  cd myfirst_workflow
+  ls
+```
+
+The directory contains a typical Molgenis Compute workflow structure
+
+file | description
+---------|----
+`/protocols` |  folder with bash script Protocols
+`/protocols/step1.sh` | Shell script template for the first protocol
+`/protocols/step2.sh` | Shell script template for the second protocol
+`workflow.csv` | file listing steps and parameter flow
+`workflow.defaults.csv` | default parameters for workflow.csv (optional)
+`parameters.csv` | parameters you want to run analysis on
+`header.ftl` | user extra script header (optional)
+`footer.ftl` | user extra script footer (optional)
+
+## Steps
+Take a look at `workflow.csv` generated workflow
+
+step|protocol|dependencies
+----|--------|----------
+step1|protocols/step1.sh|in=input
+step2|protocols/step2.sh|wf=workflowName;date=creationDate;strings=step1.out
+
+![image](/res/images/compute/workflow.png?raw=true, "the Workflow")
+
+The workflow consists of two steps, `step1` and `step2`.
+
+`step1` has protocol `protocols/step1.sh`, `step2` has protocol `protocols/step2.sh`
```

Comment:
I think has is fine, it's how compute talks about the protocols and steps in the logging as well.